src/assets/scripts/client/constants/gameOptionConstants.js

```diff
@@ -26,10 +29,24 @@ export const GAME_OPTION_NAMES = {
  */
 export const GAME_OPTION_VALUES = [
     {
+        name: GAME_OPTION_NAMES.THEME,
+        defaultValue: 'default',
+        description: 'Scope Theme',
+        type: 'select',
+        onChangeEvent: EVENT.SET_THEME,
+        optionList: [
+            {
+                displayLabel: 'Default',
+                value: 'DEFAULT'
+            }
+        ]
+    },
+    {
         name: GAME_OPTION_NAMES.CONTROL_METHOD,
         defaultValue: 'classic',
         description: 'Control Method',
         type: 'select',
+        onChangeEvent: null,
```

Comment:
I haven't made it to the implementation yet, but `_noop()` would be a much better choice than `null` here.  you can call `null` as a function but you can call `_noop()`.